catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GazetteerConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.codice.ddf.spatial.geocoding;
+
+public class GazetteerConstants {
+  public static final String DEFAULT_TAG = "geonames";
```

Comment:
I think we may want to go with gazetteer as the value here, or alternatively geolocation here.  geonames seems like it represents a source of the data, not what the data is being used for.

Reply:
I believe Adam used this value in his GSR code. The plan was that I would add define this constant in DDF and then the GSR code could be changed afterwards to use it.
Agree that "gazetteer" makes more sense as a value.